DigitalLearningSolutions.Data/Services/ProgressService.cs

```diff
@@ -131,5 +148,45 @@ public void UnlockProgress(int progressId)
         {
             progressDataService.UpdateCourseAdminFieldForDelegate(progressId, promptNumber, answer);
         }
+
+        public void StoreAspProgressV2(
+            int progressId,
+            int version,
+            string? progressText,
+            int tutorialId,
+            int tutorialTime,
+            int tutorialStatus
+        )
+        {
+            progressDataService.UpdateProgressDetails(
+                progressId,
+                version,
+                DateTime.UtcNow,

```

Comment:
We should be getting this via ClockService

Reply:
Done